processor/tailsamplingprocessor/metadata.yaml

```diff
@@ -5,15 +5,96 @@ status:
   class: processor
   stability:
     beta: [traces]
-  distributions:
-  - contrib
+  distributions: [contrib]
   codeowners:
     active: [jpkrohling]
 
 tests:
   config:
-  goleak:
-    ignore:
-      top:
-        # See https://github.com/census-instrumentation/opencensus-go/issues/1191 for more information.
-        - "go.opencensus.io/stats/view.(*worker).start"
+
+telemetry:
+  metrics:
+    processor_tail_sampling_sampling_decision_latency:
+      description: Latency (in microseconds) of a given sampling policy
+      unit: µs
+      enabled: true
+      histogram:
+        value_type: int
+        bucket_boundaries: [1, 2, 5, 10, 25, 50, 75, 100, 150, 200, 300, 400, 500, 750, 1000, 2000, 3000, 4000, 5000, 10000, 20000, 30000, 50000]
+
+    processor_tail_sampling_sampling_decision_timer_latency:
+      description: Latency (in microseconds) of each run of the sampling decision timer
+      unit: µs
+      enabled: true
+      histogram:
+        value_type: int
+        bucket_boundaries: [1, 2, 5, 10, 25, 50, 75, 100, 150, 200, 300, 400, 500, 750, 1000, 2000, 3000, 4000, 5000, 10000, 20000, 30000, 50000]
+
+    processor_tail_sampling_sampling_trace_removal_age:
+      description: Time (in seconds) from arrival of a new trace until its removal from memory
+      unit: s
+      enabled: true
+      histogram:
+        value_type: int
+
+    processor_tail_sampling_sampling_late_span_age:
+      description: Time (in seconds) from the sampling decision was taken and the arrival of a late span
+      unit: s
+      enabled: true
+      histogram:
+        value_type: int
+
+    processor_tail_sampling_sampling_policy_evaluation_error:
+      description: Count of sampling policy evaluation errors
+      unit: 1
```

Comment:
The [semantic conventions guidelines](https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units) suggest using annotation units (e.g. `{errors}`) here, should we follow that recommendation?

Reply:
Great idea, I added a new PR changing that.